include/mbgl/util/geo.hpp

```diff
@@ -65,12 +63,8 @@ class ProjectedMeters {
     double northing = 0;
     double easting = 0;
 
-    ProjectedMeters(double n = 0, double e = 0)
+    ProjectedMeters(double n, double e)
```

Comment:
Shouldn't `ProjectedMeters` ctor params have the same NaN/infinite checks as in `LatLng`?